aws-rds-integration/aws-rds-integration.json

```diff
@@ -0,0 +1,166 @@
+{
+  "typeName": "AWS::RDS::Integration",
+  "description": "An example resource schema demonstrating some basic constructs and validation rules.",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-rpdk.git",
+  "properties": {
+    "IntegrationName": {
+      "description": "The name of the integration.",
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 64
+    },
+    "Tags": {
+      "type": "array",
+      "maxItems": 50,
+      "uniqueItems": true,
+      "insertionOrder": false,
+      "description": "An array of key-value pairs to apply to this resource.",
+      "items": {
+        "$ref": "#/definitions/Tag"
+      }
+    },
+    "SourceArn": {
+      "type": "string",
+      "description": "The Amazon Resource Name (ARN) of the Aurora DB cluster to use as the source for replication."
+    },
+    "TargetArn": {
```

Comment:
@khebul suggested leaving it out of the schema because the service-side validation gives a more user-friendly error than the schema-based errors. LMK what you think.

Reply:
That is acceptable as well. I recommended it because I have observed that other parameters include it. Typically, we do not impose patterns checks on strings for this reason.